### PR TITLE
v0.3.0

### PR DESCRIPTION
**Pull request by krystian8207**
* Add new filter of type `"query"` that allows to configure complex filtering rules with `queryBuilder` package.
* Add filter-focused `.print_filter` method responsible for printing filter values when calling `sum_up` on cohort.

### CODE REVIEW

File: examples/query-filter.R [hunk 1]
```diff
@@ -0,0 +1,49 @@
+library(queryBuilder)
```
Comment (galachad) [inline examples/query-filter.R:1]: Can we use `examples/query-filter.R` as a high-level test.

**Review by galachad — APPROVED** [review]
LGTM, small suggestion

File: DESCRIPTION [hunk 2]
```diff
@@ -26,7 +26,8 @@ Imports:
     glue,
     ggplot2,
     rlang (>= 1.0),
-    formatR
+    formatR,
+    queryBuilder
```
[the remark this answers is not in the document]
Comment (krystian8207) [reply DESCRIPTION]: done

File: examples/query-filter.R (hunk 1 above)
Comment (krystian8207) [reply examples/query-filter.R:1]: Added tests for query filter - not fully based on the example but related.

**Pull request merged**